components/cv_form/nested_row.tsx

```diff
@@ -40,6 +40,11 @@ export default function NestedRow({
                       />
                     )}
                   />
+                  <ErrorMessage
+                    className="w-full text-red-600"
+                    name={`[${outerArray}][${outerIndex}][${innerArray}].${itemIndex}`}
+                    component="span"
+                  />
```

Comment:
This is not displaying an error if we omit the technologies when adding projects. Any ideas why?